docs/REST API/rest-api.md

```diff
@@ -9048,6 +9048,116 @@ Content-Type: application/octet-stream
 
 `Status: 200`
 
+### Modify package
+
+_Available in Fleet Premium._
+
+Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
+
+
+`PATCH /api/v1/fleet/software/package/{titleID}`
```

Comment:
note we may want to consider using `titles` here instead, as after you upload a package it becomes a title.
 
```suggestion
`PATCH /api/v1/fleet/software/titles/{titleID}`
```